examples/Example_embed_shift_template_experiment.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    experiment_id_colname = params['metadata_experiment_colname']
</pre><p>can you make these names match? that was a huge source of confusion/accounting for me when trying to run this stuff.</p><p> </p><p>there are two otheres as well -- <code>sample_id_colname = params['metadata_sample_colname']</code> and  <code>num_simulated_experiments = params['num_simulated']</code>. I think <code>num_simulated</code> is probably fine as is tho.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/ponyo/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>